hangman_file.js

```diff
@@ -0,0 +1,74 @@
+
+
+
+  let word = words[Math.floor(Math.random() * words.length)];
+  let newArray = [];
+  let wrongGuess=[]
+  let remainingTries = 6;
+  for (let i = 0; i < word.length; i++) {
+    newArray[i] = "_";
+  }
+  hiddenWord.innerText = newArray.join(" ");
+
+  form.addEventListener("submit", () => {
+    event.preventDefault();
+
+    let form = document.querySelector("form");
+      console.log(word)
+
+    for (let i = 0; i < word.length; i++) {
```

Comment:
There's glitch that every time a wrong letter is typed, it decreases lives by number of words. It is happening because of the loop. It's definitely a glitch and you can fix by rearranging your code where not every thing belongs under for loop.